repl/repl-service.js

```diff
@@ -1,30 +1,34 @@
 var REPL_PORT = parseInt(process.env.REPL_PORT || process.argv[2] || 10001)
 var BASES = (process.env.BASES || process.argv[3] || '').split(',')
 
-require('seneca')({
+var repl = require('seneca-repl');
+
+var seneca = require('seneca')({
   tag: 'repl',
   log: 'silent',
-  legacy: { logging: false },
-  internal: { logger: require('seneca-demo-logger') },
   debug: {short_logs:true}
 })
-  .use('mesh',{
-    tag: null, // ensures membership of all tagged meshes
-    bases: BASES,
-    make_entry: function( entry ) {
-      if( 'wo' === entry.tag$ ) {
-        return {
-          route: JSON.stringify(entry.route),
-          host: entry.host,
-          port: entry.port,
-          identifier: entry.identifier$
-        }
+.use('demo-logger')
+.use('mesh',{
+  tag: null, // ensures membership of all tagged meshes
+  bases: BASES,
+  make_entry: function( entry ) {
+    if( 'wo' === entry.tag$ ) {
+      return {
+        route: JSON.stringify(entry.route),
+        host: entry.host,
+        port: entry.port,
+        identifier: entry.identifier$
       }
     }
-  })
-  .repl({
+  }
+})
+.use(repl)
+.ready(function () {
+  seneca.repl({
```

Comment:
add an issue to repl to note that these settings should be made via the plugin options, rather than a decoration on the main seneca object